pkg/endpoint/endpoint.go

```diff
@@ -272,7 +272,7 @@ type Endpoint struct {
 	realizedRedirects map[string]uint16
 
 	// BPFConfigMap provides access to the endpoint's BPF configuration.
-	bpfConfigMap *bpfconfig.EndpointConfigMap
+	ConfigMap *bpfconfig.EndpointConfigMap
```

Comment:
I'm ok with the rename but keep the field private or it will be stored and read on endpoint restore.

Reply:
Lets do minimal changes seeing we have to backport this. Will revert name change and dup Close() hook.